src/opera/parser/tosca/v_1_3/requirement_assignment.py

```diff
@@ -25,3 +25,9 @@ def normalize(cls, yaml_node):
         if isinstance(yaml_node.value, str):
             return Node({Node("node"): yaml_node})
         return yaml_node
+
+    @classmethod
+    def validate(cls, yaml_node):
+        super().validate(yaml_node)
+        if "relationship" in yaml_node.bare and not isinstance(yaml_node.bare["relationship"], str):
+            cls.abort("Expected a relationship type name as a 'relationship' value.", yaml_node.loc)
```

Comment:
Here, we are expecting a relationship template name, not type.

Reply:
Expecting type was changed to expecting template.